pkg/util/oc/oc_struct.go

```diff
@@ -421,41 +421,45 @@ func (o OC) GetYaml(t test.TestHelper, ns, kind, name string, checks ...common.C
 	return val
 }
 
-func (o OC) GetJson(t test.TestHelper, ns, kind, name string, checks ...common.CheckFunc) string {
+func (o OC) GetJson(t test.TestHelper, ns, kind, name, jsonPath string) string {
 	t.T().Helper()
-	var val string
+	var jsonString string
 	o.withKubeconfig(t, func() {
 		t.T().Helper()
-		val = shell.Execute(t, fmt.Sprintf("oc %s get %s/%s -ojson", nsFlag(ns), kind, name), checks...)
+		if jsonPath == "" {
+			jsonString = shell.Execute(t, fmt.Sprintf("oc %s get %s %s -o json", nsFlag(ns), kind, name))
+		} else {
+			jsonString = shell.Execute(t, fmt.Sprintf("oc %s get %s %s -o jsonpath='%s'", nsFlag(ns), kind, name, jsonPath))
```

Comment:
What if `jsonPath` contains single quotes?

Reply:
I'll change this

Reply:
I added a scape for single quotes